src/posix.mak

```diff
@@ -290,6 +278,8 @@ endif
 endif
 
 
+######## DMD frontend source files
```

Comment:
I read source code quite often through a portrait-oriented monitor.  I think the headers are an improvement, though I'd prefer something more visually significant like is done at the bottom
```Make
################################################################################
# DDoc documentation generation
################################################################################
```
Or maybe something like:
```Make
# DDoc documentation generation
################################################################################
```
Regardless, it would be nice if they were all consistently the same.  But, I don't want to start a bikeshedding discussion over it.  This PR is still an improvement, as it stands IMO.

Reply:
Hehe I actually like the more significant lines a lot more as well, I just wanted to keep this PR as minimal as possible to avoid potential controversies and allow a quick merge.

Reply:
You could also get rid of the ...
```Make
######################################################
```
lines with no documentation.  They add nothing but noise, IMO.  Perhaps add some text, or remove them.

Reply:
Yes, good idea. I will do so, but in a separate PR ;-)